init-tools.sh

```diff
@@ -157,7 +157,7 @@ if [ ! -e $__INIT_TOOLS_DONE_MARKER ]; then
     Tools-Override/crossgen.sh $__scriptpath/Tools
 
     echo "Making all .sh files executable under Tools."
-    ls $__scriptpath/Tools/*.sh | xargs chmod +x
+    find $__scriptpath/Tools/ -name "*.sh" -exec chmod +x {} \;
```

Comment:
Yeah, some shells support `**` as recursive but some don't (like the default macOS shell). We can also dynamically generate some levels of `*/*/*.sh`, or hard code them in (rather than just hard-coding direct dirs).